### PR TITLE
AvalancheGo@v1.7.10 Dependency

### PR DESCRIPTION
**Pull request by patrick-ogrady**

### CODE REVIEW

File: cmd/simulator/go.mod [hunk 1]
```diff
@@ -11,7 +11,7 @@ require (
 
 require (
 	github.com/StackExchange/wmi v0.0.0-20180116203802-5d049714c4a6 // indirect
-	github.com/ava-labs/avalanchego v1.7.10-rc.0 // indirect
+	github.com/ava-labs/avalanchego v1.7.10-rc.2 // indirect
```
Comment (aaronbuchwald) [inline cmd/simulator/go.mod:14]: Can we use v1.7.10?

File: plugin/main.go [hunk 2]
```diff
@@ -30,13 +28,5 @@ func main() {
...
-	plugin.Serve(&plugin.ServeConfig{
-		HandshakeConfig: rpcchainvm.Handshake,
-		Plugins: map[string]plugin.Plugin{
-			"vm": rpcchainvm.New(&evm.VM{}),
-		},
-
-		// A non-nil value here enables gRPC serving for this plugin...
-		GRPCServer: plugin.DefaultGRPCServer,
-	})
+	rpcchainvm.Serve(&evm.VM{})
```
Comment (aaronbuchwald) [inline plugin/main.go:31]: ❤️

File: scripts/versions.sh [hunk 3]
```diff
@@ -3,4 +3,4 @@
 # Set up the versions to be used
 subnet_evm_version=${SUBNET_EVM_VERSION:-'v0.2.1'}
 # Don't export them as they're used in the context of other calls
-avalanche_version=${AVALANCHE_VERSION:-'v1.7.10-rc.0'}
+avalanche_version=${AVALANCHE_VERSION:-'v1.7.10-rc.2'}
```
Comment (aaronbuchwald) [inline scripts/versions.sh:6]: Can we use v1.7.10?

**Review by aaronbuchwald — APPROVED** [review]
LGTM

**Review by StephenButtolph — APPROVED** [review]

**Pull request merged**